### PR TITLE
Automate prombench for GKE

### PR DESCRIPTION
**Pull request by sipian**
PR for modifying prombench into GKE

### CODE REVIEW

File: provider/gke/gke.go [hunk 1]
```diff
@@ -416,7 +436,26 @@ func (c *GKE) serviceApply(resource runtime.Object) {
...
+				return nil
+			})
+			if err != nil {
+				log.Fatalf("error while reading directory%v", err)
+			}
+		} else {
+			fileList = append(fileList, file)
+		}
+	}
+
```
Comment (krasi-georgiev) [inline provider/gke/gke.go]: since you use this above as well might be better to extract to separate func

File: cmd/prombench/main.go [hunk 2]
```diff
@@ -20,9 +20,9 @@ func main() {
 		Action(g.NewGKEClient).
 		Action(g.ConfigParse)
 	k8sGKE.Flag("config", "yaml GKE config file used to create or delete the k8s cluster and nodes").
-		Default("../../config/cluster.yaml").
 		PlaceHolder("cluster.yaml").
 		Short('c').
+		// Default("../../config/cluster.yaml").
```
Comment (krasi-georgiev) [inline cmd/prombench/main.go]: I set this to the default file just for convenience.

Was there a problem with this?

Comment (sipian) [reply cmd/prombench/main.go]: It's a temporary workaround that I did.
Will revert back once PR is completed

Comment (krasi-georgiev) [reply cmd/prombench/main.go]: you can always override the default with the flag , I ma just curious why did yo have to remove it?

**Comment by sipian** [comment]
:+1:

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
 clusters
 .terraform*
 terraform*
+prombench
```
Comment (krasi-georgiev) [inline .gitignore]: I would say just remove the `.gitignore` file completely and use `.git/info/exclude` so we don't have to keep updating the .gitignore all the time. Now it is fine, but this gets annoying very quickly.

**Comment by sipian** [comment]
I had reverted back to go as was facing some issues with vgo.
Have resolved the issue now. Using vgo now

**Pull request closed without merging**